apps/webhook/src/bootstrap.ts

```diff
@@ -2,19 +2,24 @@ import './config';
 import { INestApplication } from '@nestjs/common';
 import { NestFactory } from '@nestjs/core';
 import * as Sentry from '@sentry/node';
-import { version } from '../package.json';
+import { getOTELSDK } from '@novu/application-generic';
+import * as packageJson from '../package.json';
 
 import { AppModule } from './app.module';
 
 if (process.env.SENTRY_DSN) {
   Sentry.init({
     dsn: process.env.SENTRY_DSN,
     environment: process.env.NODE_ENV,
-    release: `v${version}`,
+    release: `v${packageJson.version}`,
```

Comment:
we should also pass the version from env variables

Reply:
An environment variable can easily get forgotten or not updated, however, the package.json is the source of truth which is why I used it.

Reply:
true, but I just mentioned that because we have had issues before with this approach and we wanted to update it somewhere later on